File: Python/Funciones/is_right_truncatable_strong_harshad_prime.py
# Función usada en Project Euler #387

def is_right_truncatable_strong_harshad_prime(n: int) -> int:
    if is_prime(n) != 1:
        return 0
    elif n < 10:
        return 0
    else:
        n //= 10
        if is_strong_harshad(n) == 1 and is_right_truncatable_harshad(n) == 1:
            return 1
    return 0

File: C/Funciones/isRightTruncatableStrongHarshadPrime.c
//Función usada en Project Euler #387

int isRightTruncatableStrongHarshadPrime(long long int n){
    if (isPrime(n) != 1) return 0;
    else if (n < 10) return 0; //Menores que 10 return 0?
    else{
        n /= 10;
        if (isStrongHarshad(n) == 1 && isRightTruncatableHarshad(n) == 1) return 1;
    }
    return 0;
}
